hotness/consumers.py

```diff
@@ -220,7 +220,7 @@ def _handle_anitya_update(self, upstream, package, msg):
         # Is it new to us?
         mdapi_url = '{0}/koji/srcpkg/{1}'.format(self.mdapi_url, package)
         self.log.debug("Getting pkg info from %r" % mdapi_url)
-        r = requests.get(mdapi_url)
+        r = requests.get(mdapi_url, timeout=30)
```

Comment:
Would it make sense to have this a configuration variable? Or at least a global variable so we have it stored in one place only?